### DISCUSSION TITLE
Release 5.3.4 and 5.4.0

### DISCUSSION THREAD

**Discussion by miguel-a-calles-mba** [serverless-heaven/serverless-webpack#1550] [Team Posts]
Should we release 5.3.4 that has patched and focus on getting 5.4.0 (new features) ready to release?

5.3.4 https://github.com/serverless-heaven/serverless-webpack/milestone/32

5.4.0 https://github.com/serverless-heaven/serverless-webpack/milestone/28

**Comment by j0k3r** [comment]
Yep good idea

**Comment by miguel-a-calles-mba** [comment]
5.3.4 is now released.

Now on to 5.4.0.
